presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1378,6 +1388,57 @@ static protocol::Duration getUptime(
   return protocol::Duration(seconds, protocol::TimeUnit::SECONDS);
 }
 
+void PrestoServer::convertToVeloxPlan(
+    proxygen::HTTPMessage* message,
+    proxygen::ResponseHandler* downstream,
+    const std::vector<std::unique_ptr<folly::IOBuf>>& body) {
+  protocol::ExecutionFailureInfo failure;
```

Comment:
Move the definition closer to the variable usage.

Reply:
`failure` needs to be outside the scope of the try block. It's assigned when an exception is caught and then sent in the error response, that is why it's defined here.